daemon/graphdriver/vfs/driver.go

```diff
@@ -90,12 +99,14 @@ func (d *Driver) Get(id, mountLabel string) (string, error) {
 	return dir, nil
 }
 
+// Put return nul for the error as VFS driver has no runtime resources (e.g. mounts) to clean up.
```

Comment:
Although, something like this might sound better: `Put is a noop for vfs, since this driver has no runtime resources to clean up`.